CMakeLists.txt

```diff
@@ -222,10 +222,14 @@ if(ENABLE_UNIT_TEST OR ENABLE_INTEGRATION_TEST)
   add_library(Sourcetrail::gtest_main ALIAS gtest_main)
   target_link_libraries(
     gtest_main
-    PUBLIC GTest::gmock GTest::gtest
-    PRIVATE spdlog::spdlog)
+    PUBLIC GTest::gmock
+           GTest::gtest
+           spdlog::spdlog
+           Sourcetrail::lib_utility)
 endif()
 
+string(LENGTH "${CMAKE_SOURCE_DIR}/" SOURCE_PATH_PREFIX_LEN)
+
```

Comment:
You can add `add_definition(SOURCE_PATH_PREFIX_LEN=${SOURCE_PATH_PREFIX_LEN})` It will be define to all the project. Windows will be different so we need to check windows too.

Reply:
I don't have a Windows machine, so I already mentioned in the PR description that it's only tested on Linux.